contracts/dummy/upgradeCheck/dummyRoundFactoryV1.sol

```diff
@@ -0,0 +1,131 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity 0.8.17;
+
+import "./dummyRoundImplementationV1.sol";
+import "@openzeppelin/contracts-upgradeable/proxy/ClonesUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+
+import "../../utils/MetaPtr.sol";
+
+/**
+ * @notice Invoked by a RoundOperator to enable creation of a
+ * round by cloning the RoundImplementation contract.
+ * The factory contract emits an event anytime a round is created
+ * which can be used to derive the round registry.
+ *
+ * @dev RoundFactory is deployed once per chain and stores
+ * a reference to the deployed RoundImplementation.
+ * @dev RoundFactory uses openzeppelin Clones to reduce deploy
+ * costs and also allows uprgrading RoundContract
+ * @dev This contract is Ownable thus supports ownership transfership
+ *
+ */
+contract DummyRoundFactoryV1 is OwnableUpgradeable {
+
+  string public constant VERSION = "0.2.0";
+
+  // --- Data ---
+
+  /// @notice Address of the RoundImplementation contract
+  address public roundContract;
+
+  /// @notice Address of the protocol treasury
+  address payable public protocolTreasury;
+
+  /// @notice Protocol fee percentage
+  uint8 public protocolFeePercentage;
+
+  // --- Event ---
+
+  /// @notice Emitted when protocol fee percentage is updated
+  event ProtocolFeePercentageUpdated(uint8 protocolFeePercentage);
+
+  /// @notice Emitted when a protocol wallet address is updated
+  event ProtocolTreasuryUpdated(address protocolTreasuryAddress);
+
+    /// @notice Emitted when a Round contract is updated
+  event RoundContractUpdated(address roundAddress);
+
+  /// @notice Emitted when a new Round is created
+  event RoundCreated(address indexed roundAddress, address indexed ownedBy, address indexed roundImplementation);
+
+
+  /// @notice constructor function which ensure deployer is set as owner
+  function initialize() external initializer {
+    __Context_init_unchained();
+    __Ownable_init_unchained();
+  }
+
+  // --- Core methods ---
+
+  /**
+   * @notice Allows the owner to update the overall protocol fee percentage
+   *
+   * @param newProtocolFeePercentage New protocol fee percentage
+   */
+  function updateProtocolFeePercentage(uint8 newProtocolFeePercentage) public onlyOwner {
+
+    protocolFeePercentage = newProtocolFeePercentage;
```

Comment:
you're right, but we don't fix dummy contracts, since these are old implementations and just used to test upgradeability of the current deployed contracts